task-3/data-access/user.ts

```diff
@@ -0,0 +1,8 @@
+const { Sequelize } = require("sequelize");
+
+const sequelize = new Sequelize("USERS", "epamadmin", "securePassword123", {
```

Comment:
Completely forgot about this line.
It's a major vulnerability to expose your credentials like this, so do use env variable for them.